paket.dependencies

```diff
@@ -1,35 +1,59 @@
 source http://nuget.org/api/v2
+redirects: on
+strategy: max
 
-nuget FSharp.Compiler.Service
 nuget Fantomas
+nuget FSharp.Compiler.Service
+nuget FSharp.Core
+nuget FSharp.Management
 nuget FSharp.ViewModule.Core
-nuget FsXaml.Wpf
-nuget SourceLink.SymbolStore 0.5.0
 nuget FSharpLint.Core prerelease
-nuget VSSDK.IDE.Deploy.14
-nuget FSharp.Management
 nuget FsPickler
-nuget FSharp.Core redirects:on
+nuget FsXaml.Wpf
+nuget SourceLink.SymbolStore 0.5.0
 
+group vssdklegacy
+    source http://nuget.org/api/v2
+    framework >= net40
+    
+    nuget Microsoft.VisualStudio.Shell.Interop.8.0
+    nuget Microsoft.VisualStudio.Shell.Immutable.10.0
+    nuget VSSDK.ComponentModelHost.11
+    nuget VSSDK.DTE
+    nuget VSSDK.DTE.8
+    nuget VSSDK.DTE.10
+    nuget VSSDK.NavigateTo
+    nuget VSSDK.VSLangProj
+    
+    
+group vssdk
+    source http://nuget.org/api/v2
+    framework >= net45
+    
+    nuget Microsoft.VisualStudio.Editor
+    nuget Microsoft.VisualStudio.Language.Intellisense
```

Comment:
Thanks @jaredpar

> I wish there was a stronger way to prevent this.

This is merely preference of maintainer of this repository to not have a verbose `paket.depedencies` file, we could specify version constraints with nice constraint dsl supporting all the operators one would expect (you can see simple example: http://fsprojects.github.io/Paket/dependencies-file.html) and in any case the full resolution of locked dependencies are readable at all times, in a single place: in `paket.lock` (including transitive dependencies that we don't have to specify in `paket.dependencies`).

As you were explaining writing the qualified name in the *proj files is recommended in certain circumstances, I've open a feature request for paket: https://github.com/fsprojects/Paket/issues/1823 which will eventually add an option to have that behaviour, thanks a lot for your input!